pkg/minikube/registry/global.go

```diff
@@ -76,7 +110,13 @@ func Available() []DriverState {
 			priority = Unhealthy
 		}
 
-		sts = append(sts, DriverState{Name: d.Name, Priority: priority, State: s})
+		if vm {
```

Comment:
With go, we try to avoid if/else statements when possible, particularly in a loop. Kind of along the lines of https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

This would be clearer:

```
if vm && !IsVM(d.Name) {
  continue
}
```